### ISSUE TITLE
Works cited ends in comma instead of period for mla-new.

### ISSUE THREAD

**Issue by a-vrma** [jmclawson/biblatex-mla#20]
I'm having a problem where one specific reference ends with a comma instead of a period.
Here is a picture:
![bug](https://user-images.githubusercontent.com/29077900/56175556-aac8e780-5fc5-11e9-9f0a-46b0ba85b846.png)

The tex file:
```tex
\documentclass{article}
\usepackage[style=mla-new,hyperref=false]{biblatex}
\addbibresource{mla_bug.bib}

\begin{document}
Test \autocite[150]{king}.

\printbibliography
\end{document}
```

And the bib file:
```
@book{king,
    address = {Boston},
    series = {The {King} legacy series},
    title = {Where Do We Go from Here: Chaos or Community?},
    isbn = {978-0-8070-0067-0},
    shorttitle = {Where do we go from here},
    publisher = {Beacon Press},
    location = {Boston},
    author = {King, Jr, Martin Luther},
    year = {2010},
    origdate = {1967},
    keywords = {African Americans, Civil rights, History, Race relations, Racism, United States}
}
```

**Comment by jmclawson** [comment]
Thanks for bringing this to my attention, and with an easy-to-use MWE! I just tested it here on my end and found the same problem. 

The bug is connected to the handling of the ISBN number in your bib file. As long as you don't have the ISBN defined in your bib file, the comma is replaced with a period. 

I need to do some work to figure out how best to fix the problem for the next release, but I hope that temporary workaround (not defining the ISBN) is helpful for now.

**Comment by jmclawson** [comment]
This is fixed in the next release.

**Comment by cachandlerdev** [comment]
Since I am currently having this comma-period issue, I just wanted to double check that this is an problem with my Linux distribution using an older package, and not biblatex-mla itself. My `mla-new.bbx` file inside the texlive folder says `% mla.bbx, v1.9 2016/11/01 * compatible with biblatex 3.4` at the top. This means that I currently have `v1.9` installed, and I will not benefit from this bug fix until I upgrade to `v2.0`, right?

Edit: `tex --version` says that I am running `TeX 3.141592653 (TeX Live 2021)`

**Comment by jmclawson** [comment]
@cachandlerdev yes, you won’t see the fix until upgrading. If you’d like to create a MWE with a simple document and one bib entry showing the error on your side, I can double check that it works by typesetting it on my system.
